server/item/crossbow.go

```diff
@@ -150,6 +146,38 @@ func (c Crossbow) ReleaseCharge(releaser Releaser, tx *world.Tx, ctx *UseContext
 	return true
 }
 
+// shoot fires the crossbow's loaded projectiles.
+func (c Crossbow) shoot(releaser Releaser, tx *world.Tx, creative bool, multishot bool) {
+	rot := releaser.Rotation()
+	angles := []float64{0, -10, 10}
+
+	for _, angle := range angles {
+		if !multishot && angle != 0 {
+			continue
+		}
```

Comment:
Perhaps instead of passing `multishot` and `creative`, we can pass something like `offsetAngle` and `canPickup`?

e.g.

```go
c.shoot(releaser, tx, 0, !creative)

if multishot {
    c.shoot(releaser, tx, -10, false)
    c.shoot(releaser, tx, 10, false)
}
```